query/src/test/resources/nrt-performance-writer.xml

```diff
@@ -37,11 +37,11 @@
             -->
 
             <!-- Enable sharding on writers -->
-            <property name="index_defaults.sharding.strategy">hash</property>
-            <property name="index_defaults.sharding.number_of_shards">6</property>
+            <property name="sharding.strategy">hash</property>
+            <property name="sharding.number_of_shards">6</property>
```

Comment:
Same comment about sharding: it will be driven by the segment IDs, not the document IDs.